current-events/activity-current-events.md

```diff
@@ -0,0 +1,160 @@
+# Activity: Current Events
+
+## Goal
+
+Our goal is to create intentional space to discuss current events in technology.
+
+We want to become part of the conversation between technologists that evolves every day.
```

Comment:
"evolves every day" is describing the conversation, but has that confusing intervening clause
prefer among, since there are more than two technologists. 😁 
```suggestion
We want to become part of the ever-evolving conversation among technologists.
```